proteomic_aggregation_and_visualization/python/proteomic_aggregation_and_visualization.ipynb

```diff
@@ -0,0 +1,4004 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    
</pre><p>what is psm?  </p><p> </p><p>Consider turning line 2 (the print statement) into an assert.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microbiomedata/nmdc_notebooks/pull/120/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>